lib/couchdb.js

```diff
@@ -1276,51 +1159,42 @@ CouchDB.prototype._find = function(dbName, query, cb) {
 };
 
 /**
- * Create an index in couchdb,
- * you can specify the ddocName and indexName,
- * regarding how to create model/property index, see discussion in
- * https://github.com/strongloop/loopback-connector-cloudant/pull/153#issuecomment-307177910
- * @param {String} ddocName without prefix `_design/`
- * @param {String} indexName index name
- * @param {Array} fields we only allow single field atm,
- * so it should be a string actually
+ * A model index's naming convertion: '_design/LBModel__Foo__LBIndex__foo_index',
+ * this function returns the model prefix, default as 'LBModel'.
+ * 
+ * TBD: Do we allow customized prefix? same question for 
+ * `getIndexPropertyPrefix`, allow the flexibility would cause 
```

Comment:
> If we have a sure way to find out if there is a ddoc already defined with the prefix

Even with the current solution, I could not stop user from naming their model as `LBModel__Foo__LBIndex`, which results in interfering the indexes in model `Foo`. But with the non-configurable prefixes, we can explain our naming convention in a very clear way: 'LBModel' and 'LBIndex' are reserved words when naming a model attached to cloudant/couchdb connector.